server/core/region.go

```diff
@@ -30,12 +30,14 @@ import (
 // RegionInfo records detail region info.
 type RegionInfo struct {
 	*metapb.Region
-	Leader          *metapb.Peer
-	DownPeers       []*pdpb.PeerStats
-	PendingPeers    []*metapb.Peer
-	WrittenBytes    uint64
-	ReadBytes       uint64
-	ApproximateSize int64
+	Leader               *metapb.Peer
+	DownPeers            []*pdpb.PeerStats
+	PendingPeers         []*metapb.Peer
+	PendingLearnerPeers  []*metapb.Peer
+	CompleteLearnerPeers []*metapb.Peer
```

Comment:
Any document about how the `CompleteLearner` works? Specifically, when will a peer be added to the list and when will it be removed?